### DISCUSSION TITLE
Drawer help

### DISCUSSION THREAD

**Discussion by MaddPenguin** [Alexays/Waybar#2687] [Q&A]
Hi, this is my current MPD group: 

![prev](https://github.com/Alexays/Waybar/assets/60058928/4c7d9933-4eaf-4560-b5c4-438134bd466e)

This is the config:
```
    "group/ncmpcpp": {
        "orientation": "horizontal",
        "drawer": {
            "transition-duration": 200,
            "children-class": "battery.logo",
            "transition-left-to-right": false,
            },
        "modules": [
            "mpd",
            "mpd#prev",
            "mpd#next"
            
        ]
    },
```

Is it possible to make the controls appear on the right side? Like this:

![screenshot-20231123-204849](https://github.com/Alexays/Waybar/assets/60058928/0bcceea7-0f27-4a6e-876e-ee19078eeab4)

**Comment by Nojjia** [comment]
`"transition-left-to-right": true`
I'm 2 years late, but did it work?
